api/base/filters.py

```diff
@@ -176,6 +189,11 @@ def parse_query_params(self, query_params):
                 # Special case date(time)s to allow for ambiguous date matches
                 if isinstance(field, self.DATE_FIELDS):
                     query[field_name].extend(self._parse_date_param(field, field_name, op, value))
+                elif not isinstance(value, int) and len(value.split(',')) > 1:
```

Comment:
Well my original intent for this PR was allowing you to bulk `GET` any filterable field, including title, description, etc.  But after just discussing with @brianjgeiger, I will restrict this functionality to just the id's.  He is right that more thought will be needed to filter on some of these other fields.